data-reporting-service/src/main/resources/application.yaml

```diff
@@ -14,4 +14,4 @@ logging:
     name: PersonService.log
     path: ${DA_LOG_PATH:logs}
 server:
-  port: '8080'
+  port: '8081'
```

Comment:
Are the ports for the other services determined?

Reply:
8080 is usually used by other services as default port. So making this to 8081. 
We are free to fix whatever port as per foundations team.

Reply:
Got it. Thanks Anand!